src/gui.js

```diff
@@ -8502,6 +8502,35 @@ SpriteIconMorph.prototype.copySound = function (sound) {
     this.object.addSound(dup.audio, dup.name);
 };
 
+// SpriteIconMorph flashing
+
+SpriteIconMorph.prototype.flash = function () {
+    var previousOutlineColor = this.outlineColor.copy(),
+        previousOutline = this.outline,
+        previousState = this.userState;
+
+    this.outlineColor = SpriteMorph.prototype.highlightColor;
+    this.outline = 2;
+    this.userState = 'pressed';
+
+    this.rerender();
+
+    world.animations.push(new Animation(
```

Comment:
I'd be careful about directly referencing the "world" variable, just in case there's a page with several worlds on it at some point in the future. How about instead assigning a local var "world"?

Reply:
Oh yes. I actually forgot to assign it, but that was my intention :)